### ISSUE TITLE
Autocomplete and chips position of options on mobile

### ISSUE THREAD

**Issue by henryw14** [miguelcobain/ember-paper#950]
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/4096934/42093118-885ed546-7bab-11e8-9570-eed89bffbf9a.gif)

As you can see in the video, the positioning of the select options in the autocomplete chips dropdown (but also seen with the regular autocomplete) is more or less completely unusable. It appears to related to the keyboard and whatever the positioning computation is. 

[Angular material has the same issue](https://github.com/angular/material2/issues/6341) which I assume is due to us sharing the style sheet. Any suggestions?

**Comment by vmachacek** [comment]
+1

**Comment by pauln** [comment]
This looks as though it may be limited to Safari (iOS) - a bit of quick testing found that it seems to work fine in Chrome on both Android and iOS (though it did seem slightly less accurate/reliable on iOS than on Android), but exhibits this strange behaviour in Safari.

It looks as though a PR has been filed to (hopefully) fix this issue in angular/material2: https://github.com/angular/material2/pull/12119
I'm not sure how applicable the fix is to ember-paper, but it might at least serve as inspiration for a fix/workaround here.

**Comment by henryw14** [comment]
My current stopgap was to use the bundled version of `power-select` and `power-select-multiple` directly with the material design theme theme that power-select provides. They work flawlessly on ios safari, but the API is different and the visual appeal of the autocomplete/chips in this library is honestly better IMO so I'll switch back whenever a fix for this can be developed

**Comment by Ravenstine** [comment]
I don't think this is limited to mobile.  I'm having the same issue on desktop in Chrome.

Forgive my terrible styling: https://vimeo.com/283323849

As you can see, it seems limited to a certain "depth" of the page, probably the height of the viewport.  And it also causes this bizarre effect where the bottom of the page scrolls infinitely.  I've not added special styling or addons that would conflict.  I even turned off all other styles besides ember-paper and it does the same thing.

**Comment by Ravenstine** [comment]
@xomaczar I see no difference when I try that.  Using 1.0.0-beta.8.
